src/main/scala/ru/d4team/chat/Main.scala

```diff
@@ -1,26 +1,22 @@
 package ru.d4team.chat
 
-import ru.d4team.chat.config.{AppConfig, PostgresConfig}
-import ru.d4team.chat.db.PgDBMigrator
+import ru.d4team.chat.config.AppConfig
+import ru.d4team.chat.db.DBMigrator
 import zio._
 import zio.logging.backend.SLF4J
-import zio.sql.HikariConnectionPool
 
 object Main extends ZIOAppDefault {
 
   private val loggerLayer = Runtime.removeDefaultLoggers >>> SLF4J.slf4j
 
   private val httpApp = ZIO.never.exitCode
 
-  private val mainApp = AppConfig.logConfig *> PgDBMigrator.migrate *> httpApp
+  private val mainApp = AppConfig.logConfig *> DBMigrator.migrate *> httpApp
 
   override def run: IO[Any, ExitCode] = mainApp.provide(
     // Config
     AppConfig.allConfigs,
     AppConfig.live,
-    // DB
-    PostgresConfig.hikariConnectionPoolConfig,
-    HikariConnectionPool.live,
```

Comment:
Don't forget to return it back

Reply:
@OlegKoshelev 
If I return it, the error will appear due to the next problem:
```
──── ZLAYER WARNING ──────────────────────────────────────────────────

 You have provided more than is required.
 You may remove the following 2 layers:
   
   1. PostgresConfig.hikariConnectionPoolConfig
   2. HikariConnectionPool.live
  
──────────────────────────────────────────────────────────────────────


  override def run: IO[Any, ExitCode] = mainApp.provide(
```

Reply:
Ok, maybe just comment them)